test/community/assortativity.jl

```diff
@@ -0,0 +1,23 @@
+using Random, Statistics
+
+@testset "Assortativity" begin
+    # Test definition of assortativity as Pearson correlation coefficient
+    # between excess of degrees
+    @testset "Small graphs" for n = 5:10
+        @test assortativity(wheel_graph(n)) ≈ -1/3
```

Comment:
Probably want `@test @inferred` here (and in the other tests).

Reply:
No problem, although I'm curious: why should `@inferred` be used? What type ambiguity might be expected?

Reply:
In our tests, we try to have at least one `@inferred` test. In the past it has been used to find some inconsistencies when you cycle through graph types; most recently it's been used to show that Julia nightly has some regressions.

Reply:
Done.